api/python/provisioner/salt.py

```diff
@@ -1478,7 +1478,8 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
                 return
 
         try:
-            self._yum_rollback()
+            for target, txn_id in self._last_txn_ids.items():
+                self.yum_rollback(txn_id, target)
         except Exception as exc:
             self._rollback_error = exc
```

Comment:
Print a log statement in this exception (although an unchanged code)

Reply:
Addressed